public/index.html

```diff
@@ -26,6 +26,7 @@
     -->
     <title>React App</title>
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
importing css can be done in React too using `import`
but this is ok too

Reply:
Yes, you are right and before I used to do that but I read somewhere that it is more performant to do in this way.
Although I don't think there is a real difference in small projects like mine.